### PR TITLE
taxonomy: misc fixes

### PR DESCRIPTION
**Pull request by teolemon**
### What
taxonomy: misc fixes

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -42905,10 +42995,17 @@ sv:naturligt fläderbärsextrakt
...
 da:hyldbærsirup
 de:Holunderbeersirup
 es:jarabe de sauco
 fi:seljamarjasiirappi
+=======
+da:hyldebærsirup
+de:Holunderbeersirup
+es:jarabe de sauco
+fi:seljanmarjasiirappi
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/ingredients.txt:43008]:
```suggestion
```

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -42905,10 +42995,17 @@ sv:naturligt fläderbärsextrakt
 <en:elderberry
 en:elderberry syrup
 bg:сироп от бъз
+<<<<<<< Updated upstream
```
Comment (teolemon) [inline taxonomies/ingredients.txt:42998]:
```suggestion
```

File: taxonomies/data_quality.txt [hunk 3]
```diff
@@ -3444,11 +3452,11 @@ en:rich in vitamin a label claim but vitamin a below 0.0016
 description:en:In EU, claim that a food is a source of vitamins and/or minerals may only be made where the product contains at least a significant amount as defined in the Annex to Directive 90/496/EEC. See: https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX:31990L0496
 
 <en:Data quality warnings
-en:vitamin d source label claim but vitamin d below 5e-06
+en:vitamin-d-source-label-claim-but-vitamin-d-below-5e-06
 description:en:In EU, claim that a food is a source of vitamins and/or minerals may only be made where the product contains at least a significant amount as defined in the Annex to Directive 90/496/EEC. See: https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX:31990L0496
 
 <en:Data quality warnings
-en:rich in vitamin d label claim but vitamin d below 1e-05
+en:rich-in-vitamin-d-label-claim-but-vitamin-d-below-1e-05
```
Comment (teolemon) [inline taxonomies/data_quality.txt:3459]:
```suggestion
en:Rich in Vitamin D label claim but Vitamin D below 1e 05
```

File: taxonomies/data_quality.txt [hunk 4]
```diff
@@ -3380,31 +3388,31 @@ en:energy free label claim but energy above limitation
...
 description:en:In EU, a claim that a food is fat free may only be made where the product contains no more than 0,5 g of fat per 100 g or 100 ml. See: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02006R1924-20141213
 
 <en:Data quality warnings
-en:low saturated fat label claim but fat above limitation
+en:low saturated-fat label claim but fat above limitation
 description:en:In EU, a claim that a food is fat free may only be made if the sum of saturated fatty acids and trans-fatty acids in the product does not exceed 1,5 g per 100 g for solids or 0,75 g/100 ml for liquids and in either case the sum of saturated fatty acids and trans-fatty acids must not provide more than 10 % of energy. See: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02006R1924-20141213
 
 <en:Data quality warnings
-en:saturated fat free label claim but fat above 0.1
+en:saturated-fat-free-label-claim-but-fat-above 0.1
```
Comment (teolemon) [inline taxonomies/data_quality.txt:3403]:
```suggestion
en:Saturated fat free label claim but fat above 0.1
```

File: taxonomies/data_quality.txt [hunk 5]
```diff
@@ -3380,31 +3388,31 @@ en:energy free label claim but energy above limitation
...
 description:en:In EU, a claim that a food is low sugar may only be made where the product contains no more than 5 g of sugars per 100 g for solids or 2,5 g of sugars per 100 ml for liquids. See: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02006R1924-20141213
 
 <en:Data quality warnings
-en:sugar free label claim but sugar above limitation
+en:Sugar free label claim but sugar above limitation
 description:en:In EU, a claim that a food is sugar-free may only be made where the product contains no more than 0,5 g of sugars per 100 g or 100 ml. See: https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02006R1924-20141213
 
 <en:Data quality warnings
-en:no added sugar label claim but contains added sugar
+en:no-added-sugar-label-claim-but-contains-added-sugar
```
Comment (teolemon) [inline taxonomies/data_quality.txt:3415]:
```suggestion
en:No added sugar label claim but contains added sugar
```

File: taxonomies/ingredients.txt [hunk 6]
```diff
@@ -42905,10 +42995,17 @@ sv:naturligt fläderbärsextrakt
...
 de:Holunderbeersirup
 es:jarabe de sauco
 fi:seljamarjasiirappi
+=======
+da:hyldebærsirup
+de:Holunderbeersirup
+es:jarabe de sauco
+fi:seljanmarjasiirappi
+>>>>>>> Stashed changes
 fr:sirop de sureau
```
Comment (teolemon) [inline taxonomies/ingredients.txt:43009]:
```suggestion
bg:сироп от бъз
da:hyldbærsirup
de:Holunderbeersirup
es:jarabe de sauco
fi:seljamarjasiirappi
fr:sirop de sureau
```

File: taxonomies/ingredients_processing.txt [hunk 7]
```diff
@@ -2357,7 +2438,10 @@ sv:rökt
 
 #<en:smoked
 en:double smoked
-es:doble ahumado,doble ahumada,doble ahumados,doble ahumadas
```
Comment (benbenben2) [inline taxonomies/ingredients_processing.txt:2360]: Are you sure you want to remove some?

File: taxonomies/ingredients.txt [hunk 8]
```diff
@@ -15,6 +15,8 @@ synonyms:en:hydrolysed, hydrolyzed
 
 synonyms:en:pasteurised, pasteurized
 
+synonyms:fr:de production biologique, issu de l'agriculture biologique,
```
Comment (benbenben2) [inline taxonomies/ingredients.txt:18]: Do you have examples?

Remember that synonyms in the taxonomies are for words that are part of the ingredient (not the whole ingredient)

**Review by benbenben2 — APPROVED** [review]
Looks good

See 2 comments

File: taxonomies/ingredients_processing.txt [hunk 9]
```diff
@@ -2357,7 +2438,10 @@ sv:rökt
...
+es:doble ahumado, doblemente ahumado
```
Comment (teolemon) [inline taxonomies/ingredients_processing.txt:2441]:
```suggestion
es:doble ahumado, doble ahumada,doble ahumados,doble ahumadas,doblemente ahumado,
```

File: taxonomies/ingredients_processing.txt (hunk 7 above)
Comment (teolemon) [reply taxonomies/ingredients_processing.txt:2360]: fixed

**Pull request merged**